src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1696,8 +1696,6 @@ namespace ReportSizingManager {
 							}
 							SupFanNum = PrimaryAirSystem( CurSysNum ).SupFanNum;
```

Comment:
Forgot to include the link before to the code in [HVACUnitarySystem](https://github.com/NREL/EnergyPlus/blob/173bb58bcc95ec005c91946ecee8f573224904fd/src/EnergyPlus/HVACUnitarySystem.cc#L1886-L1888) that currently sets PrimaryAirLoop.SupFanNum.  This just needs to check the fan type and then set PrimaryAirLoop.supFanModelTypeEnum and either supFanVecIndex or SupFanNum.  Will push that up shortly.